infrastructure/template.yaml

```
@@ -22,6 +22,10 @@ Parameters:
     Type: String
     Default: common-cri-api
     Description: The name of the stack containing the common CRI lambdas/infra
+  TXMAStackName:
```

```suggestion
  TxmaStackName:
```
Nitpick but we use this name in other repos, so for consistency we can use the same name here